doc/build-unix.md

```diff
@@ -2,7 +2,8 @@ UNIX BUILD NOTES
 ====================
 Some notes on how to build Dash Core in Unix.
 
-(for OpenBSD specific instructions, see [build-openbsd.md](build-openbsd.md))
+(For BSD specific instructions, see [build-openbsd.md](build-openbsd.md) and/or
```

Comment:
Side note: we do not have `build-openbsd.md`, our docs mentioning it are misleading. We have a couple of sections for OpenBSD in `build-unix.md` instead.